test/models/post_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
+  def setup
+  	@post = posts(:one)
+    @user = users(:one)
+    
+  end	
+  test 'post has many comments' do
```

Comment:
Pls run me through on how we were able to create comments for our post data.

Reply:
Comments fixtures =)